csharp/src/Ice/StreamWrapper.cs

```diff
@@ -18,24 +18,22 @@ namespace IceInternal
     // For output streams, we use a different strategy:
     // Slice sequences are encoded on the wire as a count of elements, followed
     // by the sequence contents. For arbitrary .NET classes, we do not know how
-    // big the sequence that is eventually written will be. To avoid excessive
-    // data copying, this class maintains a private _bytes array of 254 bytes and,
-    // initially, writes data into that array. If more than 254 bytes end up being
-    // written, we write a dummy sequence size of 255 (which occupies five bytes
-    // on the wire) into the stream and, once this class is disposed, patch
-    // that size to match the actual size. Otherwise, if the _bytes buffer contains
+    // big the sequence will be. To avoid excessive data copying, this class maintains
```

Comment:
We should use the full 120-column width for these comments (and all comments in general).